src/testing/testing_suite.nit

```diff
@@ -375,11 +372,17 @@ class TestCase
 end
 
 redef class MMethodDef
-	# TODO use annotations?
 
-	# Is the method a test_method?
-	# i.e. begins with "test_"
-	private fun is_test: Bool do return name.has_prefix("test_")
+	private fun has_annotation(mbuilder: ModelBuilder, name: String): Bool do
+		var anode = mbuilder.mentity2node(self)
+		if not anode isa APropdef then return false
+		return anode.get_single_annotation(name, mbuilder) != null
+	end
```

Comment:
Same comment about the model/AST subordination, but the correct way seems unnecessary complex :(